### PR TITLE
Translated install.md in French

### PR DESCRIPTION
**Pull request by Theomat**
- The translation of "repositories" into "répertoire" is not quite right but i didn't find a better word.( "dépot" might be better ? )
- The translation of "packaging policy" into "politique de construction des paquets" might not be the best one.

### CODE REVIEW

**Comment by Chris00** [comment]
Yes « dépôt » is better.

File: site/docs/install.fr.md [hunk 1]
```diff
@@ -53,13 +54,154 @@ $ opam init
...
+
+* [OCPWin](http://www.typerex.org/ocpwin.html) est un fichier binaire contenant la
+  distribution d'OCaml pour Windows. Quelques fonctionnalités supplémentaires
+  ont été ajoutés à OCaml pour mieux supporter Windows, telles que l'auto-détection
+  de MSVC et le support de MSVC 9.0, 10.0, 11.0, 12.0 and 14.0 (VS2015).
+
+* [OCaml pour Windows](https://fdopen.github.io/opam-repository-mingw/)
+  fournit une version expérimentale du répertoire d'OPAM et d'OPAM pour Windows.
+  
+Ici sont quelques alternatives :
```
Comment (Chris00) [inline site/docs/install.fr.md]: « Voici quelques »

File: site/docs/install.fr.md [hunk 2]
```diff
@@ -53,13 +54,154 @@ $ opam init
...
+
+```bash
+apt-get install ocaml-nox # Si vous ne voulez pas le support de X11 
+apt-get install ocaml
+```
+
+Les autres paquets Unbuntu liés à OCaml sont
+[listés ici](http://packages.ubuntu.com/search?keywords=ocaml) (en anglais).
+
+Les répertoirs officiels d'Ubuntu sont souvent en retard par rapport aux
```
Comment (Chris00) [inline site/docs/install.fr.md]: « répertoirs » typo.  « dépôts » ?

File: site/docs/install.fr.md [hunk 3]
```diff
@@ -53,13 +54,154 @@ $ opam init
...
+## [FreeBSD](https://www.freebsd.org/)
 
-## Utilisation d'un autre gestionnaire de paquets
+```bash
+pkg_add -r ocaml-nox11 # Si vous ne voulez pas le support de X11 
+pkg_add -r ocaml
+```
+## Linux
+
+La plupart des distributions Linux permttent d'installer OCaml et/ou OPAM avec le gestionnaire de paquet système.
```
Comment (Chris00) [inline site/docs/install.fr.md:65]: « paquets », pluriel ?

**Comment by Chris00** [comment]
Thanks for your work!

**Comment by Theomat** [comment]
You're welcome. I have updated the file according to your comments.

**Comment by pw374** [comment]
Thanks!

**Pull request merged**